packages/frontend/core/src/components/workspace/index.tsx

```diff
@@ -83,6 +90,7 @@ export const ToolContainer = (props: PropsWithChildren): ReactElement => {
       className={clsx(toolStyle, {
         trash: inTrash,
       })}
+      aria-disabled={isSidePanelOpen && inChatPanel}
```

Comment:
Why add it here? The ai island has been controlled and hidden in another place.

https://github.com/toeverything/AFFiNE/blob/c7217ed443a86f02161cd0e621144928fae87270/packages/frontend/core/src/components/pure/ai-island/index.tsx#L34

Reply:
<img width="324" alt="image" src="https://github.com/user-attachments/assets/a0bb8ac7-8508-4a79-b27c-f3d2fd23a8d2">

move the island wrapper is not enough, it's parent still blocking events